lib/mina/runner/pretty.rb

```diff
@@ -8,14 +8,14 @@ class Pretty
       attr_reader :script
 
       def initialize(script)
-        @script = script
+        @script = Shellwords.shellsplit(script)
```

Comment:
This was a bug I introduced in https://github.com/mina-deploy/mina/pull/686, E2E tests caught it.